shared/fs/common/security-prefs.js

```diff
@@ -0,0 +1,123 @@
+// @flow
+import React from 'react'
+import {BackButton, Box, Text} from '../../common-adapters'
+import {globalStyles, globalColors, platformStyles} from '../../styles'
+import {fileUIName} from '../../constants/platform'
+
+type Props = {
+  back: () => void,
+  openSecurityPrefs: () => void,
+  needAction: boolean,
+}
+
+const InstallSecurityPrefs = (props: Props) =>
+  props.needAction ? (
+    <Box style={stylesContainer}>
+      <BackButton key="back" onClick={props.back} style={stylesClose} />
+      <Text type="HeaderBig" style={{paddingBottom: 13, paddingTop: 10}}>
+        Ghhh. Try this.
+      </Text>
+      <Text type="Body" style={{paddingBottom: 24}}>
+        Open your macOS Security & Privacy Settings and follow these steps.
+      </Text>
+      <Box style={{...globalStyles.flexBoxRow, marginRight: 20}}>
+        <Box style={{position: 'relative'}}>
+          <img
+            width={500}
+            height={437}
+            src={/* TODO: fix image sourcing */ require('../../images/install/security-preferences.png')}
+          />
+          <Box
+            style={{...styleHighlight, height: 30, left: 42, position: 'absolute', top: 350, width: 162}}
```

Comment:
why are all these things absolutely positioned?

Reply:
The rendering is merely copied from the old old (`shared/folders/install/security-prefs.desktop.js`). @chrisnojima Would you rather I fix this to not be absolutely positioned?